packages/aragon-wrapper/src/index.test.js

```diff
@@ -616,42 +593,36 @@ test('should throw if no ABI is found, when calculating the transaction path', a
   t.plan(1)
   // arrange
   const instance = new Aragon()
-  instance.permissions = Observable.create(observer => {
-    observer.next({
-      counter: {
-        add: {
-          allowedEntities: ['0x1', '0x2']
-        },
-        subtract: {
-          allowedEntities: ['0x1'],
-          manager: 'im manager'
-        }
+  instance.permissions = of({
+    counter: {
+      add: {
+        allowedEntities: ['0x1', '0x2']
+      },
+      subtract: {
+        allowedEntities: ['0x1'],
+        manager: 'im manager'
       }
-    })
-  })
-  instance.forwarders = Observable.create(observer => {
-    observer.next([
-      {
-        appId: 'forwarderA',
-        proxyAddress: '0x999'
-      }
-    ])
-  })
-  instance.apps = Observable.create(observer => {
-    observer.next([
-      {
-        appId: 'counterApp',
-        kernelAddress: '0x123',
-        abi: 'abi for counterApp',
-        proxyAddress: '0x456'
-      }, {
-        appId: 'votingApp',
-        kernelAddress: '0x123',
-        // abi: 'abi for votingApp',
-        proxyAddress: '0x789'
-      }
-    ])
+    }
   })
+  instance.forwarders = of([
+    {
+      appId: 'forwarderA',
+      proxyAddress: '0x999'
+    }
+  ])
+  instance.apps = of([
```

Comment:
Small note: `from()` and `of()` will create Observables that _end_, whereas our previous creation methods created Observables that didn't. This latter behaviour is actually more realistic most of the time; our Observables never end because Ethereum never ends.

This isn't a big deal, and I think we should always be programming Observables with the assumption that they can end anyway (if we ever handle network error states, etc).